numpy/core/tests/test_simd.py

```diff
@@ -707,7 +722,7 @@ def test_arithmetic_div(self):
         assert div == data_div
 
     def test_arithmetic_reduce_sum(self):
-        if not self._is_fp():
```

Comment:
Could you add an appropriate docsting to indicate which npyv_* macros this tests?

Reply:
docstring added.